### ISSUE TITLE
4 empty buttons on activiti app

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#946]
Within the activiti app in general there seems to be 4 empty buttons 
Unfortunately I cannot seem to see which element the WAVE tool is pointing to
N.b. the screenshot below is with no styles selected in the WAVE tool

![screen shot 2016-10-28 at 17 37 27](https://cloud.githubusercontent.com/assets/13200338/19814528/6fc282e4-9d36-11e6-8061-abdd7f9dd731.png)

**Comment by mauriziovitale** [comment]
@hashraf88  We can't fix this accessibility bug because the empty buttons are generated by the material design date picker. 
@magemello can we close this issue ?

**Comment by magemello** [comment]
@mauriziovitale , let's try with that:

.mddtp-button::after{content: 'date button'}
